packages/keyring-api/src/KeyringClient.test.ts

```diff
@@ -85,6 +85,198 @@ describe('KeyringClient', () => {
     });
   });
 
+  describe('listAccountTransactions', () => {
+    it('returns an empty list of transactions', async () => {
+      const id = '49116980-0712-4fa5-b045-e4294f1d440e';
+      const expectedResponse = {
+        data: [],
+        next: null,
+      };
+
+      mockSender.send.mockResolvedValue(expectedResponse);
+      const transactions = await keyring.listAccountTransactions(id, {
+        limit: 10,
+      });
+
+      expect(mockSender.send).toHaveBeenCalledWith({
+        jsonrpc: '2.0',
+        id: expect.any(String),
+        method: 'keyring_listAccountTransactions',
+        params: { id, pagination: { limit: 10 } },
+      });
```

Comment:
WDYT about this pattern?

```suggestion
      const pagination = { limit: 10 };
      const transactions = await keyring.listAccountTransactions(id, pagination);

      expect(mockSender.send).toHaveBeenCalledWith({
        jsonrpc: '2.0',
        id: expect.any(String),
        method: 'keyring_listAccountTransactions',
        params: { id, pagination },
      });
```